### PR TITLE
Added some (hopefully) useful documentation

### PR DESCRIPTION
**Pull request by CalebMorris**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,48 @@
...
+- transaction
+  - refund
+  - void
+
+## Dev
+
+# Adding API Endpoint
+
+Each API Endpoint needs a valition and form pair for translating the
+ user payload to cooresponding format.
```
Comment (jwdotjs) [inline README.md]: corresponding

File: README.md [hunk 2]
```diff
@@ -2,4 +2,48 @@
...
+  - remove
+- transaction
+  - refund
+  - void
+
+## Dev
+
+# Adding API Endpoint
+
+Each API Endpoint needs a valition and form pair for translating the
```
Comment (jwdotjs) [inline README.md]: valition?

**Comment by jwdotjs** [comment]
:+1:

File: README.md (hunk 2 above)
Comment (CalebMorris) [inline README.md]: validation

**Pull request merged**